windows/CommonSettings.props

```diff
@@ -0,0 +1,94 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Well, this props file is only applicable to msbuild.

The problems of not requiring explicit copy are:

1. Users are not as mindful about the settings, which vary quite significantly from team to team. 
2. Git maintenance is a nightmare:
 - Contributors modify the settings and may mistakenly commit the changes. Since it's already in the index, you cannot add it to `.gitignore`.
 - For ones who are mindful about it, keeping an uncommitted modified file is a hassle. Because they wouldn't be able to pull.

Requiring explicit copy only makes CI epsilon-more complicated. Just add one more line to the yaml file.